readme.txt

```diff
@@ -9,16 +11,60 @@ Smaily newsletter subscription plugin for WordPress
 
 == Description ==
 
-Smaily newsletter subscription plugin for WordPress.
+Smaily email marketing and automation extension plugin for Wordpress.
+Automatically generate a simple sign-up form and add new subscribers to Smaily subscribers list.
+
+= Features =
+
+**Wordpress Newsletter Subscribers**
+
+- Add new subscribers to Smaily subscribers list.
+- Select autoresponder to send automated emails.
+
+**Subscription Widget**
+
+- Option for custom advanced HTML form.
+- Functional even with NoScript users.
+- Supports sending custom form values to Smaily.
+
+= Requirements =
```

Comment:
You can remove "Requirements" section. It is already covered by `Requires PHP` and `Requires at least` properties.